### ISSUE TITLE
Bug in line 1335 in GPT_SoVITS/TTS_infer_pack /TTS.py

### ISSUE THREAD

**Issue by zddllc** [RVC-Boss/GPT-SoVITS#2688]
between 1326-1335 in GPT_SoVITS/TTS_infer_pack /TTS.py

                           # ## vits串行推理
                            for i, idx in enumerate(tqdm(idx_list)):
                                phones = batch_phones[i].unsqueeze(0).to(self.configs.device)
                                _pred_semantic = (
                                    pred_semantic_list[i][-idx:].unsqueeze(0).unsqueeze(0)
                                )  # .unsqueeze(0)#mq要多unsqueeze一次
                                audio_fragment = self.vits_model.decode(
                                        _pred_semantic, phones, refer_audio_spec, speed=speed_factor, sv_emb=sv_emb
                                    ).detach()[0, 0, :]

                            batch_audio_fragment.append(audio_fragment)  ###试试重建不带上prompt部分

we only save the last sentence result in batch, so please make "batch_audio_fragment.append(audio_fragment)" inside the loop,
otherwise we can only have the last sentence result from each batch.

Maybe many people do not change speed_factor value, so do not easy to find this bug.

**Comment by RVC-Boss** [comment]
@ChasonJiang

**Comment by ChasonJiang** [comment]
> [@ChasonJiang](https://github.com/ChasonJiang)

@RVC-Boss 已PR #2689

**Comment by RVC-Boss** [comment]
已修复，感谢提醒
